source/getting-started/managing-dependencies.md

```diff
@@ -0,0 +1,263 @@
+      'lookupFixture',
+      'raw',
+      'request',
+    ]
+  }
+});
+```
+
+To use this asset in your app, import it.
+For example, with `ic-ajax`, when to use `ic.ajax.raw`:
+
+```js
+import { raw as icAjaxRaw } from 'ic-ajax';
+//...
+icAjaxRaw( /* ... */ );
+```
+
+##### Environment Specific Assets
+
+If you need to use different assets in different environments, specify an object as the first parameter. That object's key should be the environment name, and the value should be the asset to use in that environment.
+
+```Brocfile.js
+app.import({
+  development: 'bower_components/ember/ember.js',
+  production:  'bower_components/ember/ember.prod.js'
+});
+```
+
+If you need to import an asset in one environment but not import it or any alternatives in other environments then you can wrap `app.import` in an `if` statement.
+
+```Brocfile.js
+if (app.env === 'development') {
+  app.import('vendor/ember-renderspeed/ember-renderspeed.js');
+}
+```
+
+##### Customizing a built-in Asset
+
+This is somewhat non-standard and discouraged, but suppose that due to a requirement in your application that you need to use the full version of
+Handlebars even in the production environment.  You would simply provide the path to the `EmberApp` constructor:
+
+```Brocfile.js
+var app = new EmberApp({
+  vendorFiles: {
+    'handlebars.js': {
+      production: 'bower_components/handlebars/handlebars.js'
+    }
+  }
+});
+```
+
+Alternatively, if you want to exclude the built-in asset from being
+automatically included in `vendor.js`, you can set its value to `false`:
+
+```Brocfile.js
+var app = new EmberApp({
+  vendorFiles: {
+    'handlebars.js': false
+  }
+});
+
+```
+
+_Note: The built-in assets are required dependencies needed by the environment
+to run your app. If you use the above method to specifically exclude
+some, you should still be including them in some other way._
+
+##### Test Assets
+
+You may have additional libraries that should only be included when running tests (such as qunit-bdd or sinon). These can be imported into your app in your Brocfile.js:
+
+```Brocfile.js
+var EmberApp = require('ember-cli/lib/broccoli/ember-app'),
+    isProduction = EmberApp.env() === 'production';
+
+var app = new EmberApp();
+
+if ( !isProduction ) {
+    app.import( app.bowerDirectory + '/sinonjs/sinon.js', { type: 'test' } );
+    app.import( app.bowerDirectory + '/sinon-qunit/lib/sinon-qunit.js', { type: 'test' } );
+}
+
+module.exports = app.toTree();
+```
+
+**Notes:**
+- Be sure to pass `{ type: 'test' }` as the second argument to `app.import`.  This will ensure that your libraries are compiled into the `test-support.js` file.
+
+#### Styles
+
+##### Static CSS
+
+Provide the asset path as the first argument:
+
+```Brocfile.js
+app.import('bower_components/foundation/css/foundation.css');
+```
+
+All style assets added this way will be concatenated and output as `/assets/vendor.css`.
+
+##### Dynamic Styles (SCSS, LESS, etc)
```

Comment:
Is there default support for SASS and LESS with Ember CLI? If not, then we shouldn't mention it here - leave it to the addon documentation.